src/pytestqt/qtbot.py

```diff
@@ -617,9 +668,9 @@ def captureExceptions(self):
         with capture_exceptions() as exceptions:
             yield exceptions
 
-    capture_exceptions = captureExceptions
```

Comment:
Note: removed this line becausemypy gave a warning about this also being assigned in the constructor here: https://github.com/pytest-dev/pytest-qt/blob/master/src/pytestqt/qtbot.py#L153